SecurityService.OpenIdConnect.IntegrationTests/Common/DockerHelper.cs

```diff
@@ -123,7 +248,19 @@ public async Task BeforeScenario()
             ChromeOptions options = new ChromeOptions();
             options.AddArguments("--window-size=1920,1080");
             options.AddArguments("--start-maximized");
-            options.AddArguments("--headless");
+            options.AddArguments("--disable-gpu");
+            options.AddArguments("--no-sandbox");
+            options.AddArguments("--disable-dev-shm-usage");
+            options.AddArguments("--browser.enabled_labs_experiments=same-site-by-default-cookies=2");
+            options.AddArguments("--browser.enabled_labs_experiments=cookies-without-same-site-must-be-secure=2");
+            //options.AddArguments("--headless");
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/stuart_ferguson1/SecurityService/pullRequest?prid=6144581)